server/testutil/webtester/webtester.go

```diff
@@ -440,7 +441,26 @@ func GetBazelBuildFlags(wt *WebTester, appBaseURL string, opts ...SetupPageOptio
 	for _, fn := range opts {
 		fn(wt)
 	}
-	rawBazelrc := wt.Find(`[data-header=".bazelrc"] .contents`).Text()
+
+	refetch := true
+	attempts := 1
+	var rawBazelrc string
+	for refetch && attempts <= 5 {
+		rawBazelrc = wt.Find(`[data-header=".bazelrc"] .contents`).Text()
+
+		// API key takes a moment to populate on the page, so keep reloading the text
+		// until it is fetched.
+		re := regexp.MustCompile(`build --remote_header=x-buildbuddy-api-key=(\S+)`)
```

Comment:
I think this test flakiness is actually surfacing a legitimate bug which is that we're not displaying a loading spinner here. I call it a "bug" rather than a UX issue because if the user is on a slow/flaky network connection, this content will never be displayed to the user, and they'll have no indication that it should be displayed. If we showed a loading spinner, it would indicate that we're blocked on a network call somehow. In the test, we could just wait for the loading spinner to go away, rather than doing a regex match.

Reply:
Ack - I'm going to prioritize some bazelcon related features for now, so created a ticket to track this https://github.com/buildbuddy-io/buildbuddy-internal/issues/3760

This flaky test somewhat frequently impedes the release. @jdhollen @bduffany how would you feel about merging this as is for now given that it's just a test?

Reply:
@maggie-lou I can send a fix for this today if you want to re-assign that issue to me

Reply:
🙏